### PR TITLE
Adding orbit feature for siddhi gpl extension - pmml

### PR DESCRIPTION
**Pull request by thiliA**

### CODE REVIEW

File: pmml-evaluator/1.1.16-wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>2.3.7</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            org.jpmml.evaluator.*;-split-package:=merge-first
```
Comment (suhothayan) [inline pmml-evaluator/1.1.16-wso2v1/pom.xml]: remove the ;-split-package:=merge-first its not needed

File: pmml-manager/1.1.16-wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>2.3.7</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            org.jpmml.manager.*;-split-package:=merge-first
```
Comment (suhothayan) [inline pmml-manager/1.1.16-wso2v1/pom.xml]: remove ;-split-package:=merge-first not needed here as we have only one dependancy

File: pmml-model/1.1.16-wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>2.3.7</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            org.jpmml.model.*,
+                            org.dmg.pmml.*;-split-package:=merge-first
```
Comment (suhothayan) [inline pmml-model/1.1.16-wso2v1/pom.xml]: remove ;-split-package:=merge-first as its not needed

File: pmml-schema/1.1.16-wso2v1/pom.xml [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>2.3.7</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            org.jpmml.schema.*;-split-package:=merge-first
```
Comment (suhothayan) [inline pmml-schema/1.1.16-wso2v1/pom.xml]: remove ;-split-package:=merge-first

File: pmml-evaluator/1.1.16-wso2v1/pom.xml [hunk 5]
```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (suhothayan) [inline pmml-evaluator/1.1.16-wso2v1/pom.xml]: Make all the versions to have 1.1.16.wso2v1 inserted of 1.1.16-wso2v1 and also fix other orbits to have proper versions and fix their group id

File: pmml-evaluator/1.1.16-wso2v1/pom.xml (hunk 1 above)
Comment (thiliA) [inline pmml-evaluator/1.1.16-wso2v1/pom.xml]: Added a fix in commit 330d6ad

File: pmml-manager/1.1.16-wso2v1/pom.xml (hunk 2 above)
Comment (thiliA) [inline pmml-manager/1.1.16-wso2v1/pom.xml]: Added a fix in commit 330d6ad

File: pmml-model/1.1.16-wso2v1/pom.xml (hunk 3 above)
Comment (thiliA) [inline pmml-model/1.1.16-wso2v1/pom.xml]: Added a fix in commit 330d6ad

File: pmml-schema/1.1.16-wso2v1/pom.xml (hunk 4 above)
Comment (thiliA) [inline pmml-schema/1.1.16-wso2v1/pom.xml]: Added a fix in commit 330d6ad

File: pmml-evaluator/1.1.16-wso2v1/pom.xml (hunk 5 above)
Comment (thiliA) [inline pmml-evaluator/1.1.16-wso2v1/pom.xml]: Fixed in  commit 330d6ad

File: pmml-evaluator/1.1.16.wso2v1/pom.xml [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>2.3.7</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            org.jpmml.evaluator.*
```
Comment (lasanthafdo) [inline pmml-evaluator/1.1.16.wso2v1/pom.xml:67]: @thiliA 
It is always recommended to provide export package versions as a best practice.

File: pmml-model/1.1.16.wso2v1/pom.xml [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            org.jpmml.model.*,
+                            org.dmg.pmml.*
+                        </Export-Package>
+                        <Import-Package>
+                            *;resolution:=optional
```
Comment (lasanthafdo) [inline pmml-model/1.1.16.wso2v1/pom.xml:69]: @thiliA 
Again, as a best practice, it is better to import the correct packages with versions without giving a '*;resolution:=optional' directive. Importing packages this way can lead to OSGi issues.

File: pmml-evaluator/1.1.16.wso2v1/pom.xml (hunk 6 above)
Comment (thiliA) [inline pmml-evaluator/1.1.16.wso2v1/pom.xml:67]: @lasanthafdo 
Thanks. Fixed version for all possible packages in PR: https://github.com/wso2-gpl/orbit/pull/2

File: pmml-model/1.1.16.wso2v1/pom.xml (hunk 7 above)
Comment (thiliA) [inline pmml-model/1.1.16.wso2v1/pom.xml:69]: @lasanthafdo 
Thanks. Fixed version for all possible packages in PR: https://github.com/wso2-gpl/orbit/pull/2

**Pull request merged**